kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -34,6 +34,7 @@ grCollectionsPanel.controller('GrCollectionsPanelCtrl', [
     }).catch(() => ctrl.error = true);
 
     ctrl.selectedImages$ = selectedImages$;
+    ctrl.selectedCollections = selectedCollections || [];
```

Comment:
Maybe, I put it in because sometimes it is `undefined` which then causes a type error when `.some` is called in: 

`ctrl.isSelected = ctrl.selectedCollections.some(col => {
        return angular.equals(col, ctrl.node.data.content.path);
    });`

Does that count? Or is there a better way to deal with that in `ctrl.isSelected`.